demos/common/python/models/utils.py

```diff
@@ -42,6 +42,22 @@ def __init__(self, xmin, ymin, xmax, ymax, score, id, landmarks_x, landmarks_y):
             self.landmarks.append((x, y))
 
 
+class InputTransform:
+    def __init__(self, reverse_input_channels, mean_values, scale_values):
+        self.reverse_input_channels = reverse_input_channels
+        self.mean_values = np.array(mean_values, dtype=np.float32)
+        self.scale_values = np.array(scale_values, dtype=np.float32)
+        self.is_trivial = np.array_equal(self.mean_values, [0., 0., 0.]) and \
+                          np.array_equal(self.scale_values, [1., 1., 1.]) and \
+                          not self.reverse_input_channels
+
+    def apply(self, inputs):
```

Comment:
I prefer `__call__(self, inputs):` here. It allows using `lambda x: x` in model ctor or anywhere else to set none transforms. And it would be called in model::preprocess as:
```python
data = self.input_transfrom(data)
```

Reply:
Done